curvefs/src/metaserver/fsused_manager.cpp

```diff
@@ -0,0 +1,91 @@
+/*
+ *  Copyright (c) 2023 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#include <unordered_map>
+#include "curvefs/src/metaserver/fsused_manager.h"
+
+namespace curvefs {
+namespace metaserver {
+
+void FsUsedManager::ApplyFsUsedDeltas() {
+    std::deque<FsUsedDelta> fsUsedDeltaDirty;
+    {
+        std::unique_lock<bthread::Mutex> lock_guard(dirtyLock_);
+        fsUsedDeltaDirty = std::move(fsUsedDeltas_);
```

Comment:
yes, it is ok

Reply:
I think it may be more appropriate to persist this increment to the mds side. Otherwise, every time it is restarted, the changes in the window period of all fs in the entire cluster will be lost. Of course, this is just my opinion